src/View/util/QVTKRenderWindowInteractor.py

```diff
@@ -0,0 +1,491 @@
+import vtkmodules.qt
```

Comment:
Just curious about this file. It seems to have been copied from VTK's source, right? Why not just include this file instead of copying it, were some modifications needed so it would work?

Reply:
When we first started using VTK, the file required some modification to works with PySide6. It seems that VTK has been updated to work with PySide6 now. Thank you for bringing it up.

Reply:
I just had a look myself and saw that VTK had been updated, but it seems as though it hasn't been updated on PyPi (so `pip install vtk` won't give you the newest version with PySide6 support). I think for now it's best to leave this in here, however since it's been copied from vtk and modified you will need to include the licence - just as a comment at the top of this file should be enough.

The licence file is here: https://gitlab.kitware.com/vtk/vtk/-/blob/master/Copyright.txt

Reply:
@Tamookk @Regnum771 pip hasn't released the new version of vtk which includes Pyside6 support yet. I think we should stick to this workaround for now.

Reply:
We will test the new version to see if it actually works. We'll include the copyright on top of the file as well. Thank you.

Reply:
Leave what's there for now, just add the copyright.

What is currently in this PR will need to stay until pip has updated with the new version of VTK. We don't know when this will be, and what is in this PR works perfectly, so there's no need to test the new version for this PR.

Reply:
For `DicomView3D.py`, lines 8, 66, 67, and 71 are still a little too long, and you still need to add `:param QCloseEvent: ...` in the docstring under the `closeEvent` function. Good work adding the licence and with the other lines though!

Reply:
Can you please have a review again?

Reply:
Perfect! Great work